### DISCUSSION TITLE
How to know about the workflow's running status?

### DISCUSSION THREAD

**Discussion by wpz850398226** [Netflix/conductor#2783] [Q&A] [answered]
Please do not create issues for questions. Use the [discussion forum](https://github.com/Netflix/conductor/discussions) instead!
When I start a new workflow,it returns only ID.I don't know whether workflow ran successfully or not.If it ran failed or stoped at a wait task,I can only know which node is failing or stoped by constant polling,do I?Is there other ways for workflow to actively push messages when it failing or stoped at a wait task?Please,thanks!

**Comment by v1r3n** [comment, marked as answer]
Currently, the only way to get this by polling for the status since the workflows can be long running.
You can setup a failure workflow that will get started if the workflow fails or times out if that is the use case.

**Comment by wpz850398226** [comment]
ok, I got it, thank you very much!

**Comment by apanicker-nflx** [comment]
>Is there other ways for workflow to actively push messages when it failing

Please check out the section about [Workflow Notifications](https://netflix.github.io/conductor/configuration/workflowdef/#workflow-notifications) in the documentation. You can extend the notification interface to publish an event to a system of your choice.
